model/vm_host.rb

```diff
@@ -196,6 +196,15 @@ def download_firmware(version_x64: nil, version_arm64: nil, sha256_x64: nil, sha
     Strand.create_with_id(schedule: Time.now, prog: "DownloadFirmware", label: "start", stack: [{subject_id: id, version: version, sha256: sha256}])
   end
 
+  # Introduced for downloading cloud hypervisor via REPL.
+  def download_cloud_hypervisor(version_x64: nil, version_arm64: nil, sha256_ch_bin_x64: nil, sha256_ch_bin_arm64: nil, sha256_ch_remote_x64: nil, sha256_ch_remote_arm64: nil)
+    version, sha256_ch_bin, sha256_ch_remote = (arch == "x64") ? [version_x64, sha256_ch_bin_x64, sha256_ch_remote_x64] : [version_arm64, sha256_ch_bin_arm64, sha256_ch_remote_arm64]
+    raise ArgumentError, "No version provided" if version.nil?
+    raise ArgumentError, "No SHA-256 digest provided for cloud-hypervisor" if sha256_ch_bin.nil?
+    raise ArgumentError, "No SHA-256 digest provided for ch-remote" if sha256_ch_remote.nil?
+    Strand.create_with_id(schedule: Time.now, prog: "DownloadCloudHypervisor", label: "start", stack: [{subject_id: id, version: version, sha256_ch_bin: sha256_ch_bin, sha256_ch_remote: sha256_ch_remote}])
```

Comment:
Aside, not for this patch, long standing problem: it's rather gross we don't have a `Prog::DownloadCloudHypervisor.start` type of thing that gets rid of some of the free-handing of the record we see here.